modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-impl/src/main/java/com/liferay/dynamic/data/mapping/data/provider/internal/DDMDataProviderInvokeCommand.java

```diff
@@ -43,6 +43,9 @@ public DDMDataProviderInvokeCommand(
 				HystrixCommandProperties.Setter().
 					withExecutionTimeoutInMilliseconds(
 						_getTimeout(ddmRESTDataProviderSettings))
+			).andCommandPropertiesDefaults(
+			   HystrixCommandProperties.Setter()
+					.withExecutionIsolationSemaphoreMaxConcurrentRequests(20)
 			));
```

Comment:
@rijema please reuse the existing `andCommandPropertiesDefaults`:
```
).andCommandPropertiesDefaults(
	HystrixCommandProperties.Setter().
		withExecutionIsolationSemaphoreMaxConcurrentRequests(20)
		.withExecutionTimeoutInMilliseconds(
			_getTimeout(ddmRESTDataProviderSettings))
));
```

Reply:
Ok. I will chain it.